docs/_static/niscope_usage.inc

```diff
@@ -21,41 +21,8 @@ The following is a basic example of using the **niscope** module to open a sessi
         # Find all record number 3
         rec3 = [wfm for wfm in waveforms if wfm.record == 3]
 
-The waveform returned from `fetch <niscope/class.html#fetch>`_ is a flat list of Python objects
-
-    - Attributes:
-
-        -  **relative_initial_x** (float) the time (in seconds) from the trigger to the first sample in the fetched waveform
-        -  **absolute_initial_x** (float) timestamp (in seconds) of the first fetched sample. This timestamp is comparable between records and acquisitions; devices that do not support this parameter use 0 for this output.
-        -  **x_increment** (float) the time between points in the acquired waveform in seconds
-        -  **channel** (str) channel name this waveform was acquired from
-        -  **record** (int) record number of this waveform
-        -  **gain** (float) the gain factor of the given channel; useful for scaling binary data with the following formula:
-
-                voltage = binary data * gain factor + offset
-
-        -  **offset** (float) the offset factor of the given channel; useful for scaling binary data with the following formula:
-
-                voltage = binary data * gain factor + offset
-
-        - **samples** (array of float) floating point array of samples. Length will be of the actual samples acquired
-
-    - Such that all record 0 waveforms are first. For example, with a channel list of 0,1, you would have the following index values:
-
-        - index 0 = record 0, channel 0
-        - index 1 = record 0, channel 1
-        - index 2 = record 1, channel 0
-        - index 3 = record 1, channel 1
-        - etc.
-
-
-If you need more performance or need to work with `SciPy <https://www.scipy.org/>`_, you can use the `fetch_into()` method instead of `fetch()`. This
-method takes an already allocated `numpy <http://www.numpy.org/>`_ array and puts the acquired samples in it. Data types supported:
-
-    - `numpy.float64`
-    - `numpy.int8`
-    - `numpy.in16`
-    - `numpy.int32`
+If you need more performance or need to work with `SciPy <https://www.scipy.org/>`_, you can use the **fetch_into**() method instead of **fetch**(). This
+method takes an already allocated `numpy <http://www.numpy.org/>`_ array and puts the acquired samples in it.
 
 .. code-block:: python
```

Comment:
I wasn't certain whether we should delete this whole thing. The API documentation really isn't good enough to easily determine how to actually use fetch_into and we don't use it in our examples. Without this section, I would have a hard time using it. I'd probably have to look for usage in our tests.

Reply:
We actually do use it in the fetch_forever example, but that usage is even more advanced. I created a fetch_into example and deleted the codeblock.